cmd/gpu_plugin/README.md

```diff
@@ -84,10 +84,19 @@ Successfully tagged intel/intel-gpu-plugin:devel
 
 ### Deploy plugin DaemonSet
 
-You can then use the example DaemonSet YAML file provided to deploy the plugin.
+You can then use the [example DaemonSet YAML](../../deployments/gpu_plugin/base/intel-gpu-plugin.yaml)
+file provided to deploy the plugin.
 
 ```bash
-$ kubectl create -f ./deployments/gpu_plugin/gpu_plugin.yaml
+$ kubectl apply -k deployments/gpu_plugin
+daemonset.apps/intel-gpu-plugin created
+```
+
+If your cluster runs [Node Feature Discovery](https://github.com/kubernetes-sigs/node-feature-discovery),
+you can deploy the device plugin DaemonSet only on nodes with Intel GPU:
```

Comment:
Given you added the link to the YAML file in the previous paragraph (thanks!), maybe you can link out here to the add-nodeselector yaml file?

Reply:
I also wonder if we should mention `kustomization` here - I'm not sure how common kustomize use is yet, so we might want to call it out to be clear that is in use to do the modification/specialisation of the yaml.

Reply:
Added link to the kustomization.
Added note on using the default kustomization and what it does (deploys the yaml as is...).